src/ripple/rpc/README.md

```diff
@@ -1,3 +1,72 @@
-# RPC
+# How to use RPC coroutines.
```

Comment:
Titles and section headers don't end with periods, and each word is capitalized except for articles.